assembly/src/chunkTypes.ts

```diff
@@ -31,8 +29,8 @@ export class IHDR {
     IHDR.COLOR_TYPES_TO_BIT_DEPTHS.set(3, [1, 2, 4, 8])
     IHDR.COLOR_TYPES_TO_BIT_DEPTHS.set(4, [8, 16])
     IHDR.COLOR_TYPES_TO_BIT_DEPTHS.set(6, [8, 16])
-    
-    const bitDepths: StaticArray<u8> = IHDR.COLOR_TYPES_TO_BIT_DEPTHS.get(colorType);
+
+    const bitDepths = IHDR.COLOR_TYPES_TO_BIT_DEPTHS.get(colorType);
```

Comment:
actually i wonder if using `switch` or `map` would be faster. I mean, we have only 5 branches so the difference would be negligible (if any), but that's just one of my points of curiosity.